ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -184,7 +184,9 @@ if tflite2_support_is_available
   )
   message('TensorFlow2-lite version: ' + tflite2_ver)
 
-  if not cxx.check_header('flatbuffers/flatbuffers.h')
+  # check flatbuffers header
+  ## Note that tizen tflite package has its own flatbuffers header internally.
+  if build_platform != 'tizen' and not cxx.check_header('flatbuffers/flatbuffers.h')
```

Comment:
If this ok for flatbuf/flexbuf subplugins (decoders/converters)?

Reply:
It's okay, tflite and those are decoupled.